docs/source/markdown/podman-completion.1.md

```diff
@@ -28,33 +28,47 @@ The default is **false**.
 ### BASH
 `bash-completion` has to be installed on the system.
 
-To load the completion script into the current session run:\
-**source <(podman completion bash)**
+To load the completion script into the current session run:
+```
+source <(podman completion bash)
+```
 
-To make it available for all bash sessions run:\
-**podman completion -f /etc/bash_completion.d/podman bash**.
+To make it available for all bash sessions run:
+```
+podman completion -f /etc/bash_completion.d/podman bash
+```
 
 
 ### ZSH
 Shell completion needs to be already enabled in the environment. The following can be executed:\
-**echo "autoload -U compinit; compinit" >> ~/.zshrc**
```

Comment:
Did you mean to leave in the trailing backslash here and in line 48? It seems to be harmless in the generated man page, but I haven't checked the HTML. Otherwise LGTM, and thank you for your PR!

Reply:
Agreed - these are unnecessary and I've removed them :)